src/main.cpp

```diff
@@ -169,6 +174,18 @@ int WINAPI Mine_send(SOCKET s, const char* buf, int len, int flags)
     const auto ret = _ReturnAddress();
     std::ignore = ret;
 
+    auto command = buf[8];
+
+    // check for lobby specific commands
+    if (command == 0x07 || command == 0x14 || command == 0x1F || command == 0x21 || command == 0x22 || command == 0x26 || command == 0x24)
```

Comment:
Can this be broken out into a helper function, with comments of what each command is? Maybe even an ENUM

Reply:
I split this out into a helper function.